spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/HttpServiceRegistrationBean.java

```diff
@@ -68,4 +72,84 @@ public HttpServiceRegistrationBean setRoute(@NotNull Route route) {
     public HttpServiceRegistrationBean setPathPattern(@NotNull String pathPattern) {
         return setRoute(Route.builder().path(pathPattern).build());
     }
+
+    @Override
+    public @NotNull Collection<Object> getExampleRequests() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public HttpServiceRegistrationBean setExampleRequests(@NotNull Collection<Object> exampleRequests) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public HttpServiceRegistrationBean setExampleRequests(@NotNull Iterable<Object> exampleRequests) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public HttpServiceRegistrationBean setExampleRequests(Object... exampleRequests) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public HttpServiceRegistrationBean addExampleRequests(@NotNull Collection<Object> exampleRequests) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public HttpServiceRegistrationBean addExampleRequests(@NotNull Iterable<Object> exampleRequests) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public HttpServiceRegistrationBean addExampleRequests(Object... exampleRequests) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public HttpServiceRegistrationBean addExampleRequest(@NotNull Object exampleRequest) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public @NotNull Collection<HttpHeaders> getExampleHeaders() {
+        throw new UnsupportedOperationException();
```

Comment:
Perhaps we don't need to override this because it will return an empty list anyway?